src/relay/src/createRequestHandler.js

```diff
@@ -18,18 +19,17 @@ type Sink = {|
 |};
 
 export default function createRequestHandler(customFetcher: (...args: $ReadOnlyArray<any>) => any) {
-  function cleanup() {
-    // noop, do anything here (called after sink.complete)
-  }
-
   return function handleRequest(
     requestNode: RequestNode,
     variables: Variables,
     cacheConfig: CacheConfig,
     uploadables: ?Uploadables,
   ) {
+    const controller = new AbortController();
+    const signal = controller.signal;
+
     return Observable.create((sink: Sink) => {
-      customFetcher(requestNode, variables, uploadables)
```

Comment:
Should we add a note in the `relay/readme.md` about how to use the fetch abort function

Reply:
I'd skip it for now. We are explicitly not recommending to write custom fetchers but rather use the one which we provide and that one solves it by design.